app/routes.py

```diff
@@ -1,2 +1,202 @@
-from flask import Blueprint
+from werkzeug.datastructures import Authorization
+from app import db
+from app.models.task import Task
+from flask import Blueprint, request, jsonify, make_response
+from datetime import datetime
+import os
+import requests
 
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("/<task_id>", methods=["GET","PUT", "DELETE"])
+def get_single_task(task_id):
+
+    task = Task.query.get(task_id)
+    # With the GET, POST and DELETE request if there is nothing we output this
+    if request == None or task == None:
+        return jsonify(None), 404
```

Comment:
👍 this works great! One thing we could do to shorten this is:

```suggestion
    task = Task.query.get_or_404(task_id)
```

this will do your 404 check for you!